src/pages/MovieDetailsPage/MovieDetailsPage.js

```diff
@@ -30,10 +26,9 @@ const MovieDetailsPage = () => {
         </div>
         <div className="movie-details__buttons">
           <ModalInfo />
-          <Btn title='Watch Now' className="btn__watch"/>
-          <Btn title='View Info' className="btn__info"/>
+          <button className="btn__watch">Watch Now</button>
```

Comment:
I think it will be better to create generic Button component

Reply:
Now i use Button component